assembl/static2/js/app/components/debate/brightMirror/fictionsList.jsx

```diff
@@ -0,0 +1,88 @@
+// @flow
+import * as React from 'react';
+import Masonry from 'react-masonry-component';
+import Animated from 'react-animated-transitions';
+
+import { Grid } from 'react-bootstrap';
+import { Translate, I18n } from 'react-redux-i18n';
+import { get } from '../../../utils/routeMap';
+
+import { getDiscussionSlug } from '../../../utils/globalFunctions';
+import FictionPreview from './fictionPreview';
+import { fictionBackgroundColors } from '../../../constants';
+
+type Post = {
```

Comment:
@cedrtang I think we should use BrightMirrorFictionFragment when rebasing with DEVAS 1726